sormas-ui/src/main/java/de/symeda/sormas/ui/task/TaskEditForm.java

```diff
@@ -331,7 +338,8 @@ private void updateByCreatingAndAssignee() {
 				discard(TaskDto.ASSIGNEE_REPLY, TaskDto.TASK_STATUS);
 			}
 
-			if (UserProvider.getCurrent().hasUserRight(editOrCreateUserRight)) {
+			UserProvider currentUser = UserProvider.getCurrent();
+			if (currentUser.hasUserRight(UserRight.TASK_CREATE) || currentUser.hasUserRight(UserRight.TASK_EDIT)) {
```

Comment:
Please use the user right in compliance with the `create` field, i.e. if we're in a create form, use TASK_CREATE, otherwise use TASK_EDIT